header.php

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title><?php bloginfo("name") ?></title>
+</head>
```

Comment:
Hello 😊 Nice work, but I think we're missing a wp_head() call here. Aint that right?

Reply:
You are right, I have added it in 😊